nixos/doc/manual/release-notes/rl-2009.section.md

```diff
@@ -460,7 +460,7 @@ When upgrading from a previous release, please be aware of the following incompa
 
 - `udpt` experienced a complete rewrite from C++ to rust. The configuration format changed from ini to toml. The new configuration documentation can be found at [the official website](https://naim94a.github.io/udpt/config.html) and example configuration is packaged in `${udpt}/share/udpt/udpt.toml`.
 
-- We now have a unified [services.xserver.displayManager.autoLogin](options.html#opt-services.xserver.displayManager.autoLogin) option interface to be used for every display-manager in NixOS.
+- We now have a unified [services.xserver.displayManager.autoLogin](options.html#opt-services.displayManager.autoLogin) option interface to be used for every display-manager in NixOS.
```

Comment:
I don't think old release notes should be changed?

Reply:
Fair point